src/main/java/com/parasoft/findings/jenkins/coverage/model/Percentage.java

```diff
@@ -101,7 +101,7 @@ public static Percentage valueOf(final String stringRepresentation) {
                 return new Percentage(numerator, denominator);
             }
         }
-        catch (NumberFormatException exception) {
+        catch (NumberFormatException exception) { // parasoft-suppress OWASP2021.A9.LGE "This is expected. Reason: This method has not been called."
```

Comment:
The `NumberFormatException` may come from `Integer.parseInt`. In such cases, the percentage calculation cannot proceed normally. Even if we don't want to throw an exception that causes the build to fail, we should still choose to log the relevant error information.